samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.m

```diff
@@ -14,351 +14,309 @@
-    if (indexPath.row == 30) {
-        cell.accessoryView =
-            [self _createLabelwithNumberOfLines:2 AdjustFontSizeToFitWidth:NO LineBreakMode:UILineBreakModeWordWrap MinimumFontSize:0.0f];
-    }
-    if (indexPath.row == 31) {
-        cell.accessoryView =
-            [self _createLabelwithNumberOfLines:0 AdjustFontSizeToFitWidth:NO LineBreakMode:UILineBreakModeWordWrap MinimumFontSize:0.0f];
-    }
-    if (indexPath.row == 32) {
-        cell.accessoryView =
-            [self _createLabelwithNumberOfLines:3 AdjustFontSizeToFitWidth:NO LineBreakMode:UILineBreakModeWordWrap MinimumFontSize:0.0f];
-    }
+    self.minimumScaleFactor.delegate = self;
+    self.minimumScaleFactor.keyboardType = UIKeyboardTypeDecimalPad;
+    [self.minimumScaleFactor addTarget:self action:@selector(textFieldEditDidEnd:) forControlEvents:UIControlEventEditingDidEnd];
 
-    if (indexPath.row == 33) {
-        cell.accessoryView = [self _createLabelwithNumberOfLines:2
-                                        AdjustFontSizeToFitWidth:YES
-                                                   LineBreakMode:UILineBreakModeTailTruncation
-                                              MinimumScaleFactor:0.0f];
-    }
-    if (indexPath.row == 34) {
-        cell.accessoryView = [self _createLabelwithNumberOfLines:2
-                                        AdjustFontSizeToFitWidth:YES
-                                                   LineBreakMode:UILineBreakModeTailTruncation
-                                              MinimumScaleFactor:0.1];
-    }
-    if (indexPath.row == 35) {
-        cell.accessoryView = [self _createLabelwithNumberOfLines:2
-                                        AdjustFontSizeToFitWidth:YES
-                                                   LineBreakMode:UILineBreakModeTailTruncation
-                                              MinimumScaleFactor:0.9f];
-    }
+    [self.fontSize addTarget:self action:@selector(fontSizeChanged:) forControlEvents:UIControlEventValueChanged];
+    self.fontSize.continuous = YES;
+
+    [self.autoShrink addTarget:self action:@selector(autoShrinkChanged:) forControlEvents:UIControlEventValueChanged];
+
+    [self.start addTarget:self action:@selector(startTests:) forControlEvents:UIControlEventTouchUpInside];
+    [self.logResult addTarget:self action:@selector(copyTestResult:) forControlEvents:UIControlEventTouchUpInside];
+}
+
+- (void)copyTestResult:(UIButton*)sender {
+    UIPasteboard* pasteBoard = [UIPasteboard generalPasteboard];
+    NSString* logResult = self.log.text;
+    [pasteBoard setValue:logResult forPasteboardType:(NSString*)kUTTypePlainText];
+}
 
-    return cell;
+- (void)startTests:(UIButton*)sender {
+    // reset the log
+    _eventId = 0;
+
+    __weak UILabelViewController* weakSelf = self;
+    dispatch_queue_t g = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0);
+    dispatch_async(g, ^{
```

Comment:
Worth a comment explaining that - assume new team member will not have the context.

---
In reply to: [105032490](https://github.com/Microsoft/WinObjC/pull/2178#discussion_r105032490) [](ancestors = 105032490,104797832)